tests/compat/test_gecko50.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+
+from helper import CompatTestCase
+from validator.compat import FX50_DEFINITION
+
+
+class TestFX50Compat(CompatTestCase):
+    """Test that compatibility tests for Gecko 50 are properly executed."""
+
+    VERSION = FX50_DEFINITION
+
+    def test_nsIX509Cert_removed(self):
+        """https://github.com/mozilla/addons-server/issues/3408"""
+        expected = (
+            'The methods getUsagesArray, requestUsagesArrayAsync, and '
+            'getUsagesString are no longer supported.')
+
+        script = '''
+            var certdb = Components.classes["@mozilla.org/security/x509certdb;1"].
+                getService(Components.interfaces.nsIX509CertDB);
+            var cert = certdb.findCertByDBKey(
+                tgt.parentNode.firstChild.getAttribute("display"),
```

Comment:
Ah, I assume we need `QueryInterface` because _Spidermonkey_ can't resolve `tgt`.